docs/manual/java/gettingstarted/WhatsNext.md

```diff
@@ -0,0 +1,14 @@
+# What's Next?
+
+After working through "Getting started with Lagom", follow the path that make the most sense to you:
+
+* Learn how to integrate with Eclipse:
+    * [[Import an existing Maven project into Eclipse|EclipseMavenInt]] 
+    * [[Create a new Eclipse project using the Lagom Maven archetype|EclipseMavenNew]]
+    * [[Import an existing sbt build into Eclipse|EclipseSbt]]
+    
+* Learn how to intgrate with Intellij:
+    * [[Import an existing Maven project into Intellij|intellijMaven]]
+    * [[Import an existing sbt build into Intellij|IntellijSbtJava]]
+    
+* [[Learn about Lagom core concepts|CoreConcepts]]
```

Comment:
This could be solved in CSS or in MD processing. The issue is the padding of a `<p>` tag which seems to be added in very special situations related to single-item `<ul>`. This issue is bigger than I anticipated and can't tackle it now. I've created https://github.com/lagom/lagom.github.io/issues/32 with my tests and findings.